.ci/jobs/apm-server-benckmarks-mbp.yml

```
@@ -0,0 +1,45 @@
+---
+- job:
+    name: apm-server/benchmarks
+    display-name: APM Server Benchmarks
+    description: APM Server Benchmarks
```

Can we add an `apmbench` identifier, that way we know how it compares to the other `Benchmark` job identifier.

```suggestion
    display-name: APM Server Benchmarks (apmbench)
    description: APM Server Benchmarks (apmbench)
```